plugincontainer/examples/container/Dockerfile

```diff
@@ -7,14 +7,15 @@ COPY go-plugin-counter /bin/go-plugin-counter
 
 ENTRYPOINT [ "/bin/go-plugin-counter" ]
 
-FROM docker.mirror.hashicorp.services/ubuntu as nonroot
+FROM docker.mirror.hashicorp.services/ubuntu@sha256:2e863c44b718727c860746568e1d54afd13b2fa71b160f5cd9058fc436217b30 as nonroot
```

Comment:
Can we add a comment that states which tag this digest corresponds to?

Reply:
Done, thanks!